pages/content/amp-dev/documentation/components/reference/amp-3d-gltf@ar.md

```diff
@@ -0,0 +1,121 @@
+---
+$category@: media
+formats:
```

Comment:
AFAIK formats and category are redundant in the translated files. Can you check, and if true, remove them from the front matter?

Reply:
They are not redundant. The formats are used for filtering in the sidebar by website, email, ad and stories and the categories are used for filtering and sorting on the overview pages.